container/os-autoinst_dev/Dockerfile

```diff
@@ -1,12 +1,15 @@
 #!BuildTag: os-autoinst_dev
-FROM opensuse/tumbleweed
+FROM opensuse/leap:15.3
 
 # Define environment variable
 ENV NAME openQA test environment
 ENV LANG en_US.UTF-8
 
 # install system-user-qemu already here to workaround bsc#1182451
-RUN zypper ar -f -G 'http://download.opensuse.org/repositories/devel:/openQA/openSUSE_Tumbleweed' devel_openqa && zypper -n in system-user-qemu && zypper clean
+RUN zypper ar -f -G 'http://download.opensuse.org/repositories/devel:/openQA/openSUSE_Tumbleweed' devel_openqa && \
```

Comment:
This TW line looks inconsistent. Did you test it?

Reply:
This diff isn't up to date. I can try to build the container locally, though.